portfolio/src/main/webapp/style.css

```diff
@@ -4,6 +4,42 @@
   width: 650px;
 }
 
+title {
+    text-align: center;
+}
+
 #greeting-container {
   margin-top: 20px;
 }
+p1 {
+    font-family: 'Times New Roman', Times, serif;
+    font-size: 20px;
+    
+  }
+
+  p2 {
+    font-family: 'Times New Roman', Times, serif;
+    font-size: 20px;
+     
+  }
+
+
+
+  #niceback {
```

Comment:
I'm confused by this variable name.

Reply:
Apologies I was trying to add a background but deleted it.